api/v1/annotations/cilium/annotations.go

```diff
@@ -28,4 +28,7 @@ const (
 	// bypassing all VLANs.
 	// e.g., k8sd/v1alpha1/cilium/vlan-bpf-bypass="4001,4002"
 	AnnotationVLANBPFBypass = "k8sd/v1alpha1/cilium/vlan-bpf-bypass"
+
+	// Enable the Cilium SCTP feature.
+	AnnotationEnableSCTP = "k8sd/v1alpha1/cilium/enable-sctp"
```

Comment:
Can we mimic the helm flag of `sctp.enabled`, so something like `sctp-enabled` ?

Reply:
Good point, updated.